browser/about_flags.cc

```diff
@@ -178,6 +179,11 @@ constexpr char kBraveGoogleSignInPermissionName[] =
 constexpr char kBraveGoogleSignInPermissionDescription[] =
     "Enable permissioning access to legacy Google Sign-In";
 
+constexpr char kBraveLocalhostAccessPermissionName[] =
+    "Enable Localhost access Permission Prompt";
```

Comment:
nit: looks like we're using the lowercase version of `localhost` everywhere else.